env_source/ubuntufocalfossagpu_SOURCE

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
filename: `env_source/ubuntufocalgpu_SOURCE`

Reply:
Yes, this is already in the pipeline and locally changed. This needs also an adjustment on ansible repo.

I just try to run the gpu file in the ubuntu focal (basic) and check about the pip/python issue